kafka-rest/src/main/java/io/confluent/kafkarest/entities/ConsumerInstanceConfig.java

```diff
@@ -63,96 +79,7 @@ public ConsumerInstanceConfig(
       @Nullable Integer responseMinBytes,
       @Nullable Integer requestWaitMs
   ) {
-    this.id = id;
-    this.name = name;
-    this.format = Objects.requireNonNull(format);
-    this.responseMinBytes = responseMinBytes;
-    this.requestWaitMs = requestWaitMs;
-    this.autoOffsetReset = autoOffsetReset;
-    this.autoCommitEnable = autoCommitEnable;
-  }
-
-  @Nullable
-  public String getId() {
-    return id;
-  }
-
-  @Nullable
-  public String getName() {
-    return name;
-  }
-
-  public EmbeddedFormat getFormat() {
-    return format;
-  }
-
-  @Nullable
-  public String getAutoOffsetReset() {
-    return autoOffsetReset;
-  }
-
-  @Nullable
-  public String getAutoCommitEnable() {
-    return autoCommitEnable;
-  }
-
-  @Nullable
-  public Integer getResponseMinBytes() {
-    return this.responseMinBytes;
-  }
-
-  @Nullable
-  public Integer getRequestWaitMs() {
-    return this.requestWaitMs;
-  }
-
-  public Properties toProperties() {
-    Properties properties = new Properties();
-    if (responseMinBytes != null) {
-      properties.setProperty(
-          KafkaRestConfig.PROXY_FETCH_MIN_BYTES_CONFIG, responseMinBytes.toString());
-    }
-    if (requestWaitMs != null) {
-      properties.setProperty(
-          KafkaRestConfig.CONSUMER_REQUEST_TIMEOUT_MS_CONFIG, requestWaitMs.toString());
-    }
-    return properties;
-  }
-
-  @Override
-  public boolean equals(Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
-    ConsumerInstanceConfig that = (ConsumerInstanceConfig) o;
-    return Objects.equals(id, that.id)
-        && Objects.equals(name, that.name)
-        && format == that.format
-        && Objects.equals(autoOffsetReset, that.autoOffsetReset)
-        && Objects.equals(autoCommitEnable, that.autoCommitEnable)
-        && Objects.equals(responseMinBytes, that.responseMinBytes)
-        && Objects.equals(requestWaitMs, that.requestWaitMs);
-  }
-
-  @Override
-  public int hashCode() {
-    return Objects.hash(
+    return new AutoValue_ConsumerInstanceConfig(
         id, name, format, autoOffsetReset, autoCommitEnable, responseMinBytes, requestWaitMs);
   }
-
-  @Override
-  public String toString() {
```

Comment:
What's the effect of a change like this on logging?

Reply:
AutoValue will generate reasonably similar toStrings for these entities. Most of these entities, and their toStrings were added as part of 5.5 preview. I would not expect people to be depending on them.